admin/dist/mac/bundleFrameworks.sh

```diff
@@ -159,6 +159,8 @@ plugins="imageformats sqldrivers bearer"
 for plugin in $plugins; do
     if [ -d /Developer/Applications/Qt/plugins/ ]; then
         pluginDir=/Developer/Applications/Qt/plugins
+    elif [ -d /usr/local/lib/qt4/plugins/ ]; then
+        pluginDir=/usr/local/lib/qt4/plugins/
```

Comment:
I feel like this change needs a comment explaining that this is the recommended location for qt4 when installed through Homebrew.